elements/pf-button/pf-button.css

```diff
@@ -570,7 +570,7 @@ button:hover {
 :host([block]) button {
   display: flex;
   width: 100%;
-  justify-content: var(--pf-c-button--JustifyContent, center);
```

Comment:
why this change?

Reply:
I removed this and opened that separate PR to add the CSS var, but opting for a `part` on the button instead.